examples/hosts/app-integration/external-data/src/model/containerCode.ts

```diff
@@ -67,6 +67,6 @@ export class TaskListContainerRuntimeFactory extends ModelContainerRuntimeFactor
                 taskList.importExternalData();
             }
         });
-        return new AppModel(taskList, container);
+        return new AppModel(taskList, container, runtime);
```

Comment:
Something to think about (I don't have an opinion yet) - it's important to resolve any async stuff to inject into the AppModel (like retrieving the TaskList) but beyond that what should live in the createModel method vs. the AppModel constructor?  Should the runtime.on("signal", ...) handler also move into the constructor now?

Reply:
That's an interesting thought. Moving it to AppModel is probably cleaner since then it's clear that the container doesn't define any of the behaviour on receiving the signal, but rather the app itself.